src/app/[locale]/class-based/[id]/assignments/[assignmentId]/details/components/SubmissionCard.tsx

```
@@ -69,18 +76,36 @@ export function SubmissionCard({ submission, attachments, onSubmit, loading }: P
+                            key={i}
+                            title={'link'}
+                            url={u}
+                            onRemove={() => setUrls(urls.filter((_, idx) => idx !== i))}
+                        />
+                    ))}
                     {attachments.map((attachment) => (
                         <AttachmentItem key={attachment.attachmentId} attachment={attachment} />
                     ))}
+                    {urlAttachments.map((url) => (
+                        <UrlAttachmentItem url={url} />
+                    ))}
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)